lib/agetpass.c

```diff
@@ -24,11 +24,6 @@
 #endif /* WITH_LIBBSD */
 
 
-#if !defined(PASS_MAX)
-#define PASS_MAX  BUFSIZ - 1
```

Comment:
You can remove `#include <stdio.h>` from this file now, I think.

Reply:
Yes, I can do that. I thought about it.

But in fact moving `#include <stdio.h>` to defines.h means that this include can be removed from about 70 files.

I'm not sure how to proceed with that. Extra commit? PR? What do you suggest?

Reply:
So far I added `stdio.h` removal from this file. For the others I seek a recommendation.

Reply:
> But in fact moving `#include <stdio.h>` to defines.h means that this include can be removed from about 70 files.

Don't do that, please.  :)

See:

-  <https://include-what-you-use.org/>
-  <https://google.github.io/styleguide/cppguide.html#Include_What_You_Use>

   > Do not rely on transitive inclusions.